build.gradle

```diff
@@ -36,7 +36,7 @@ jar {
     baseName = 'eiffel-remrem-semantics'
     // If this version is updated please update it in Event.java 
     // com.ericsson.eiffel.remrem.semantics.events
```

Comment:
Sure Vali. We are working with semantics so for the next release we will make sure that this doesnot come with it.